examples/api-routes-middleware/pages/index.js

```diff
@@ -1,12 +1,12 @@
-import fetch from 'isomorphic-unfetch'
+import useSWR from 'swr'
 
-const Index = ({ cookie }) => <div>{`Cookie from response: ${cookie}`}</div>
+const fetcher = url => fetch(url).then(res => res.text())
```

Comment:
Like https://github.com/zeit/next.js/pull/11383/files#r399102448
Shouldn't this deal with HTTP errors?
```suggestion
const fetcher = url => fetch(url).then(res => res.ok ? res.text() : Promise.reject(new Error(`HTTP error ${res.status}`)))
```

Reply:
There's no need to do error handling in this case, the local API route won't fail for this example, so the less complex the better.

Reply:
And in any case, if there's an error you should see it in the console and the UI will display an error message.

Reply:
it could 500.

Reply:
Any error would result in the UI showing a error, and in the case of this example it will only fail if your deployment is broken.